src/components/Header.js

```diff
@@ -0,0 +1,26 @@
+const Header = ({ title, imageSrc, isHome }) => {
+  let headerStyles;
+  let titleStyles;
+  if (isHome) {
+    headerStyles = 'bg-yellow-400 w-full text-center relative h-32 mb-32';
+    titleStyles = ' font-bold text-2xl w-1/4 mx-auto mt-4';
```

Comment:
I think we need to change the width on mobile view. It puts the title on too many lines.

![title spacing](https://user-images.githubusercontent.com/29425781/169895979-f2ba9658-4a30-45ae-8ec0-2434a21dae05.png)

Reply:
@JimeBlue will tackle this problem in another issue